commit 46771394991221321116a55de13a800cbd7b0235 (refs/changes/97/4397/2)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2018-06-21T19:26:47+00:00 (1 year, 4 months ago)
    
    TRITON-373 disable directory listing on assets zone nginx
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/boot/configure.sh b/boot/configure.sh
index 12d3327..a2445a3 100755
--- a/boot/configure.sh
+++ b/boot/configure.sh
@@ -38,7 +38,6 @@ http {
         location / {
             root   /assets;
             index  index.html index.htm;
-            autoindex on;
         }
 
         error_page   500 502 503 504  /50x.html;
